### ISSUE TITLE
`--fix` with `--format compats` raises an unexpected error

### ISSUE THREAD

**Issue by Sean0628** [textlint/textlint#979]
**What version of textlint are you using?**
```sh
>>> npx textlint -v
v12.4.0
```

**What did you expect to happen?**
Not to raise an error, and return output in compats format.

**What actually happened? Please include the actual, raw output from textlint.**
```sh
>>> npx textlint file.md --fix --format compats
Error
formatter is not a function

Stack trace
TypeError: formatter is not a function
    at /path/to/node_modules/@textlint/fixer-formatter/lib/src/index.js:120:16
    at TextFixEngine.formatResults (/path/to/node_modules/textlint/lib/src/engine/textlint-engine-core.js:243:16)
    at /path/to/node_modules/textlint/lib/src/cli.js:157:42
```

---

Once approved, I am willing to contribute to fixing this. Thanks.

**Comment by azu** [comment]
Related:

- https://github.com/textlint/textlint/issues/148
- https://github.com/textlint/textlint/issues/462

**Comment by Sean0628** [comment]
It seems that there is `compats` fixer-formatter.
cf. https://github.com/textlint/textlint/blob/master/packages/%40textlint/fixer-formatter/src/formatters/compats.ts

Correct me if I am wrong, but probably it is not returning a function at this point.
```
/path/to/node_modules/@textlint/fixer-formatter/lib/src/index.js:120:16
```

**Comment by azu** [comment]
```diff
- export function format
+ export default function
```
You are correct.
